docs/src/modules/concepts/pages/saga-patterns.adoc

```diff
@@ -1,56 +1,60 @@
+* Clear audit trail
+* Can become a coordination bottleneck
+* Tighter coupling between orchestrator and services
+a| * Decentralized control
+* Low coordination overhead
+* Services are only coupled to events
+* Increased complexity in ensuring proper failure handling
+* Harder to debug and monitor long-running flows
 
-* Decentralized control and lower coordination overhead.
-* Services operate independently, reacting to events.
+| In Akka, you can implement this pattern using the xref:java:workflows.adoc[Workflow] component. The Workflow defines each step and manages retries, timeouts, and compensating actions.
+| In Akka, you can implement this pattern by combining components such as xref:java:event-sourced-entities.adoc[Entities] and xref:java:consuming-producing.adoc[Consumers], each producing and reacting to events.
+| Example: https://github.com/akka-samples/transfer-workflow-orchestration[Funds Transfer Workflow Between Two Wallets, window="new"] 
+| Example: https://github.com/akka-samples/choreography-saga-quickstart[User Registration Service, window="new"]
 
-*Challenges:*
+|===
 
-* Increased complexity in ensuring all services handle failures correctly.
-* Harder to track and debug long-running processes without a central point of control.
+== Choosing the right pattern
 
-You can build choreography-based Sagas by combining Akka components such as Entities, Consumers, and Timers. Check out the Choreography Saga sample that can be downloaded as a link:../java/_attachments/choreography-saga-quickstart.zip[zip file].
+When selecting a Saga pattern, consider the architecture of your system and the nature of the business process.
 
-== Orchestrator-Based Saga pattern
+Use choreography-based Sagas if:
 
-In an orchestrator-based Saga, a central controller (or orchestrator) coordinates the entire process. This orchestrator manages the sequence of steps, ensuring that each transaction completes successfully or triggers compensating actions if a failure occurs. 
+* Your services are autonomous and can handle failure independently
+* You prefer low coupling and high scalability
+* You are comfortable with distributed control and eventual consistency
+* You do not require central tracking of each step
 
-In Akka, _Workflows_ implement the orchestrator-based approach. The Workflow defines each step of the process, invoking different Akka components or services. If a step fails, the orchestrator handles retries, timeouts, or compensating actions, maintaining overall control of the process.
+Use orchestrator-based Sagas if:
 
-*Advantages:*
+* Your process includes tightly coordinated steps
+* You need centralized visibility and clear state tracking
+* Retrying, error handling, and compensation must be handled consistently
+* You are fine with introducing a central coordination point
 
-* Centralized control simplifies managing the flow and handling failures.
-* Easier to track the progress and state of long-running processes.
+== How to decide
```

Comment:
might want to add that a potential downside to choreography is that its difficulty to build and maintain goes up rapidly as more components are involved in the flow, whereas with orchestration that curve isn't as steep because you have central control.